src/domain/common/constants.ts

```diff
@@ -0,0 +1,5 @@
+/**
+ * The maximum safe integer for the database.
+ * This is the maximum value that can be stored in the database for an integer column.
```

Comment:
Nit: can we add a link to a reference for this?

Reply:
Done in [e94c337](https://github.com/safe-global/safe-client-gateway/pull/1788/commits/e94c3376a1185863bb94ae76e168a2a9caf29069)